source/macos/AdaptiveCards/AdaptiveCards/Views/ACRTextView.swift

```diff
@@ -17,16 +17,6 @@ class ACRTextView: NSTextView, SelectActionHandlingProtocol {
         return NSSize(width: width, height: size.height)
     }
     
-    override func viewDidMoveToSuperview() {
```

Comment:
will this be handled in TextBlockRenderer ?

Reply:
they make this method for add height and width property. 
now we handle _height_ from **_TextBlockRenderer_** and 
_width_ from _**BaseCardElementRenderer**_ > _updateLayoutForSeparatorAndAlignment_

`114. view.setContentHuggingPriority(kFitViewHorizontalLayoutConstraintPriority, for: .horizontal)`